x-pack/plugins/rollup/public/crud_app/sections/job_list/job_table/job_table.test.js

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Pager } from '@elastic/eui';
+
+import { registerTestBed } from '../../../../../__jest__/utils';
+import { getJobs } from '../../../../../fixtures';
+import { rollupJobsStore } from '../../../store';
+import { JobTable } from './job_table';
+
+const defaultProps = {
+  jobs: [],
+  pager: new Pager(20, 10, 1),
+  filter: '',
+  sortField: '',
+  isSortAscending: false,
+  openDetailPanel: () => {},
+  closeDetailPanel: () => {},
+  filterChanged: () => {},
+  pageChanged: () => {},
+  pageSizeChanged: () => {},
+  sortChanged: () => {},
+};
+
+const initTestBed = registerTestBed(JobTable, defaultProps, rollupJobsStore);
+
+describe('<JobTable />', () => {
+  describe('table rows', () => {
+    const totalJobs = 5;
+    const jobs = getJobs(totalJobs);
+    const openDetailPanel = jest.fn();
+    const { findTestSubject } = initTestBed({ jobs, openDetailPanel });
+    const tableRows = findTestSubject('jobTableRow');
+
+    it('should create 1 table row per job', () => {
+      expect(tableRows.length).toEqual(totalJobs);
+    });
+
+    it('should create the expected 8 columns for each row', () => {
+      const expectedColumns = [
+        'id',
+        'status',
+        'indexPattern',
+        'rollupIndex',
+        'rollupDelay',
+        'dateHistogramInterval',
+        'groups',
+        'metrics'
+      ];
+
+      const tableColumns = expectedColumns
+        .map(id => {
+          const col = findTestSubject(`jobTableHeaderCell-${id}`);
+          col.__id__ = id;
+          return col;
+        })
+        .filter(col => !!col.length)
+        .map(col => col.__id__);
+
+      expect(tableColumns).toEqual(expectedColumns);
+    });
+
+    it('should set the correct job value in each row cell', () => {
+      const fields = [
```

Comment:
Can we rename this `unformattedFields` to emphasize that the fields display the original fields without doing any formatting on them?

Reply:
Not sure I understand the benefit of it. It  could be named `fieldIds` or `fieldNames` to be more correct and match the implementation.

Reply:
Well, the comment on line 75 points out that this array isn't _all_ of the fields, it's just the ones which are rendered without any special formatting. We test other fields here in different ways when they're rendered with formatting, e.g. `status` and `groups`. So we can make the code more self-explanatory by naming this array something which communicates its role within the test, which IMO would be `unformattedFields` or `fieldsRenderedWithoutFormatting` -- because it contains only the fields which aren't formatted. We could name it something else, but I think the ideal name would be one which makes the comment on line 75 unnecessary.

Reply:
I understand now what you mean 😊 Thanks for the explanation 👍